src/PrestaShopBundle/Form/Admin/Type/DeltaQuantityType.php

```diff
@@ -72,8 +72,10 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver
             ->setDefaults([
                 'delta_label' => $this->trans('Add or subtract items', 'Admin.Global'),
+                'modify_all_shops' => false,
```

Comment:
The problem with this is that it's the same option that is already used by the extension and that adds automatically the checkboxes. So this specific extra option should have a different name than `modify_all_shops` or else a checkbox mapped to the whole form type could be added

Reply:
```suggestion
                'modify_delta_for_all_shops' => false,
```